src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/TaskEntity.kt

```diff
@@ -22,7 +22,7 @@ interface TaskRepository : JpaRepository<Task, UUID> {
         CASE
           WHEN :requiredQualification = 'womens' THEN apa.is_womens_application = true
           WHEN :requiredQualification = 'pipe' THEN apa.is_pipe_application = true
-          WHEN :requiredQualification = 'emergency' THEN apa.is_emergency_application = true
+          WHEN :requiredQualification = 'emergency' THEN (apa.notice_type = 'emergency' OR apa.notice_type = 'shortNotice')
```

Comment:
I wish i'd spotted this before but we're using API generated enums in the entity model. I'd much rather we had used an internal enum. I suspect it would be a pain to change this now?

Reply:
No worries we can always change it later if it becomes a problem, it's not too difficult